sdk/core/azure-core/src/http/curl/curl_connection_private.hpp

```diff
@@ -134,6 +134,20 @@ namespace Azure { namespace Core {
        * @return `true` is the connection was shut it down; otherwise, `false`.
        */
       bool IsShutdown() const { return m_isShutDown; }
+
+      size_t m_usedCount = 0;
```

Comment:
In general, I prefer to keep data values like `m_usedCount` along with all the other data values - that way it is easier to see what data members exist in the type.

Reply:
done